src/features/defi/providers/idle/components/IdleManager/Overview/IdleEmpty.tsx

```diff
@@ -0,0 +1,218 @@
+            <Flex gap={2} alignItems='center'>
+              <AssetIcon assetId={asset?.assetId} size='sm' />
+              <Text
+                fontWeight='bold'
+                letterSpacing='-0.02em'
+                translation={['common.needAsset', { asset: asset?.name }]}
+              />
+            </Flex>
+            <Button variant='ghost' size='sm' colorScheme='blue' onClick={handleAssetBuyClick}>
+              {translate('common.buyNow')}
+            </Button>
+          </Alert>
+        )}
+        <Button size='lg' width='full' colorScheme='blue' onClick={onClick}>
+          <Text translation='common.continue' />
+        </Button>
+      </Flex>
+    )
+  }, [
+    asset?.assetId,
+    asset?.name,
+    assetSupportsBuy,
+    cryptoBalance,
+    handleAssetBuyClick,
+    onClick,
+    translate,
+  ])
+
+  const renderTags = useMemo(() => {
+    return tags?.map(tag => {
+      if (idleTagDescriptions[tag as IdleTag]) {
+        const tagDetails = idleTagDescriptions[tag as IdleTag]
+        return (
+          <Carousel key={tag} showDots options={{ loop: false, skipSnaps: false }}>
+            {tagDetails.map((page, i) => (
+              <Flex p={4} key={i} gap={4} width='full'>
+                <Flex flexDir='column' textAlign='left' gap={2}>
+                  {page.title && (
+                    <Text
+                      letterSpacing='0.012em'
+                      fontSize='lg'
+                      fontWeight='bold'
+                      translation={page.title}
+                      textShadow={`0 2px 2px var(${textShadow})`}
+                    />
+                  )}
+                  {page.description && (
+                    <Text
+                      letterSpacing='0.02em'
+                      textShadow={`0 2px 2px var(${textShadow})`}
+                      translation={page.description}
+                    />
+                  )}
+
+                  {page.bullets && (
+                    <List spacing={3}>
+                      {page.bullets.map(item => (
+                        <ListItem>
+                          <ListIcon color='green.500' as={MdCheckCircle} />
+                          {translate(item)}
+                        </ListItem>
+                      ))}
+                    </List>
+                  )}
+                </Flex>
+                {page.icon && <Flex flexBasis='150px'>{page.icon}</Flex>}
+              </Flex>
+            ))}
+          </Carousel>
+        )
+      } else return <Tag key={tag}>{tag}</Tag>
+    })
+  }, [tags, textShadow, translate])
+
+  return (
+    <SlideTransition>
+      <DefiModalContent
+        backgroundImage={bgImage}
+        backgroundSize='cover'
+        backgroundPosition='center -120px'
+        backgroundRepeat='no-repeat'
+        overflow='hidden'
+        borderRadius='2xl'
+      >
+        <EmptyOverview
+          assets={[{ icon: asset?.icon ?? '' }]}
```

Comment:
Urghhh, doesn't need to be tackled in this PR but the API of this component definitely needs some love - this is a list of asset icons, not assets